release-notes/0.8/0.8.md

```diff
@@ -0,0 +1,73 @@
+<!--
+* Copyright (c) 2017, 2018 IBM Corp. and others
+*
+* This program and the accompanying materials are made
+* available under the terms of the Eclipse Public License 2.0
+* which accompanies this distribution and is available at
+* https://www.eclipse.org/legal/epl-2.0/ or the Apache
+* License, Version 2.0 which accompanies this distribution and
+* is available at https://www.apache.org/licenses/LICENSE-2.0.
+*
+* This Source Code may also be made available under the
+* following Secondary Licenses when the conditions for such
+* availability set forth in the Eclipse Public License, v. 2.0
+* are satisfied: GNU General Public License, version 2 with
+* the GNU Classpath Exception [1] and GNU General Public
+* License, version 2 with the OpenJDK Assembly Exception [2].
+*
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] http://openjdk.java.net/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH
+* Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+# Eclipse OpenJ9 version 0.8 release notes
+
+## Supported platforms
+
+JCK compliance status for **OpenJDK with Eclipse OpenJ9** binaries:
+
+- Java 9 compliant
+- Java 8 expected to be compliant
+
+|Operating system | Platform architecture | Comments                                    |
+|-----------------|-----------------------|---------------------------------------------|
+|Linux            | x86, IBM Power, IBM Z | Windows subsytem for Linux is not supported |
+|AIX              | IBM Power             | Untested                                    |
+|Windows          | x86                   | Untested                                    |
+
+
+## Known Issues
+
+The v0.8 release contains the following known issues and limitations:
+
+#### Direct dump reader
+[Issue #378](https://github.com/eclipse/openj9/issues/378):
+The direct dump reader is not yet working, which has an impact on diagnosing problems with the VM, Garbage Collector, and the JIT. For example, [issue #968](https://github.com/eclipse/openj9/issues/968).
+
+#### Signal handling
+[Issue #54](https://github.com/eclipse/openj9/issues/54): Some signals are not being handled correctly by the VM. In particular, for CTRL-C or other signals that shutdown the VM, the shutdown hooks are not running. See [Issue #378](https://github.com/eclipse/openj9/issues/378)
+
+#### Large Java heaps
+[Issue #479](https://github.com/eclipse/openj9/issues/479): Java heaps bigger than 57 GB (or ~62 GB with command line options) are not supported.
```

Comment:
The builds from Adopt only build the "compressedrefs" binary and thus have this limitation.  OpenJ9 supports building both the compressed and non-compressed versions.  Eventually we'll enable both in the same binary, either by composing the two packages or by enabling a single binary to do both based on runtime checks.

Reply:
Added additional explanation to the Large Java Heap item.